cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +696,34 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(3)
+@pytest.mark.provider([VMwareProvider], override=True)
+@pytest.mark.usefixtures("setup_provider")
+def test_ansible_service_linked_vm(
```

Comment:
please add the automate marker for this BZ

Reply:
Updating it.